analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml

```diff
@@ -20,7 +20,7 @@ project:
     dependencies:
     - id: "GoDep::github.com/mattn/go-isatty:v0.0.12"
       linkage: "STATIC"
-    - id: "GoDep::golang.org/x/sys:05986578812163b26672dabd9b425240ae2bb0ad"
+    - id: "GoDep::golang.org/x/sys:8d3cce7afc34617998104db7c4b58c2de9e77215"
```

Comment:
Just curious, why's this needed now that all Go tests are synthetic and versions are hard-coded SHA1s?

Reply:
Does the synthetic test project really specify the dependency on `GoDep::golang.org/x/sys` explicitly (along with a SHA1)?

Reply:
No, it's an implicitly dependency that's somehow introduced. But I still wouldn't have expected it to change unless maybe the Go compiler version changes.

Reply:
This is a transitive dependency. We should monitor if this happens more often and if so replace the direct dependency with another one that does not depend on a changing package.

Reply:
A transitive dependency of what, of `go-isatty`? If so, it should be nested under `go-isatty`, no?

Reply:
GoDep does not support dependency trees, it's always a flat list.

Reply:
Oh, I wasn't aware of that. But wouldn't it be the job of our GoDep package manager implementation to still *create* a tree?